From 59c8c7740d6f236431fa792957fefe9f67611b27 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Fri, 6 Sep 2019 17:24:55 +0100
Subject: [PATCH] staging: bcm2835-codec: Allow height of 1920.

The codec is happy with video up to 1920 high if the width
is suitably reduced to stay within level limits. eg 1080x1920
is OK to decode.

Increase the height limit accordingly.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 .../staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -92,7 +92,7 @@ static const char * const components[] =
 #define MIN_W		32
 #define MIN_H		32
 #define MAX_W		1920
-#define MAX_H		1088
+#define MAX_H		1920
 #define BPL_ALIGN	32
 #define DEFAULT_WIDTH	640
 #define DEFAULT_HEIGHT	480
